website/public/style.css

```
@@ -0,0 +1,196 @@
+:root {
+  --border: #d0d7de;
+  --ink: #1f2328;
+  --muted: #59636e;
+  --accent: #0969da;
+  --surface: #f6f8fa;
+  font-size: 16px;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)